packages/mock-block-dock/package.json

```diff
@@ -24,12 +24,16 @@
     "dist"
   ],
   "scripts": {
-    "build": "yarn clean && yarn build:cjs && yarn build:esm",
-    "build:cjs": "tsc --p tsconfig.build.json --module commonjs --outDir dist/cjs ",
+    "build": "yarn clean && yarn generate-version-file && yarn build:cjs && yarn build:esm && yarn copy-files",
+    "build:cjs": "tsc --p tsconfig.build.json --module commonjs --outDir dist/cjs",
     "build:dev": "yarn clean && cross-env NODE_ENV=development webpack --mode development",
     "build:esm": "tsc --p tsconfig.build.json",
     "clean": "rimraf ./dist/",
-    "dev": "concurrently -n webpack,webpack-dev-server -c green,cyan \"yarn build:dev --watch\" \"yarn run-dev-server\"",
+    "copy-files": "yarn copy-files:cjs && yarn copy-files:esm",
+    "copy-files:cjs": "copyfiles -u 1 ./src/assets/* dist/cjs/",
+    "copy-files:esm": "copyfiles -u 1 ./src/assets/* dist/esm/",
+    "dev": "yarn generate-version-file && concurrently -n webpack,webpack-dev-server -c green,cyan \"yarn build:dev --watch\" \"yarn run-dev-server\"",
+    "generate-version-file": "node -p \"'export const MOCK_BLOCK_DOCK_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
```

Comment:
We need to display the package version in the debug UI. Pulling the version from package.json affects the build output since it also tries to bundle package.json. This creates a `version.ts` file that contains the package's version. It's called when we run `dev` and `build` so that the file is always present

Reply:
Tested `dev`, `build` and `generate-version-file` on cmd.exe and they all seem to work.
`dev` printed an error (but worked anyway): 
```
[webpack] Module not found: Error: Can't resolve './src' in 'C:\Users\lmnbe\Desktop\blockprotocol\packages\mock-block-dock'
[webpack] resolve './src' in 'C:\Users\lmnbe\Desktop\blockprotocol\packages\mock-block-dock'
[webpack]   using description file: C:\Users\lmnbe\Desktop\blockprotocol\packages\mock-block-dock\package.json (relative path: .)
[webpack]     Field 'browser' doesn't contain a valid alias configuration
```

Reply:
Thanks. @kachkaev also pointed out this issue yesterday. I will try to address in #555 or create a follow up task

Reply:
Thanks!